pkg/apis/kubermatic/v1/cluster.go

```diff
@@ -153,9 +153,13 @@ type ClusterSpec struct {
 	// Optional: ImagePullSecret references a secret with container registry credentials. This is passed to the machine-controller which sets the registry credentials on node level.
 	ImagePullSecret *corev1.SecretReference `json:"imagePullSecret,omitempty"`
 
+	// Optional: CNIPluginSettings contains the spec of the CNI plugin used by the Cluster.
 	CNIPlugin *CNIPluginSettings `json:"cniPlugin,omitempty"`
 
-	ClusterNetwork  ClusterNetworkingConfig   `json:"clusterNetwork"`
+	// Optional: ClusterNetworkingConfig specifies the different networking parameters for a cluster.
+	ClusterNetwork ClusterNetworkingConfig `json:"clusterNetwork"`
```

Comment:
The comment should mention the name of the field, not the type. In this example, you used `ClusterNetworkingConfig`, but the field is named `ClusterNetwork`. So the comment should be:

```suggestion
	// Optional: ClusterNetwork specifies different networking parameters for a cluster.
	ClusterNetwork ClusterNetworkingConfig `json:"clusterNetwork"`
```